experiments/long_runs/soil.jl

```diff
@@ -365,6 +367,7 @@ setup_and_solve_problem(; greet = true);
 #### ClimaAnalysis ####
 simdir = ClimaAnalysis.SimDir(outdir)
 short_names = ["swc", "si", "sie"]
+tmpdir = mktempdir(root_path)
```

Comment:
Maybe let's be more explicit in cleaning up the folder and do 
```suggestion
mktempdir(root_path) do tmpdir
```
(and add an `end`) at the end of this file